oci/oci.go

```diff
@@ -34,24 +34,26 @@ const (
 )
 
 // New creates a new Runtime with options provided
-func New(runtimePath string, conmonPath string, conmonEnv []string, cgroupManager string) (*Runtime, error) {
+func New(runtimePath string, runtimeHostPrivilegedPath string, conmonPath string, conmonEnv []string, cgroupManager string) (*Runtime, error) {
```

Comment:
Design question: would it make sense to pass in a struct with configuration instead? Every time a new parameter is added (as today), the method and all its callers have to be adjusted, and it's annoying for optional parameters (because all the call sites will need to be changed to pass in zero values like "")

Have we considered passing in configuration as a special struct type, like:

```go
type RuntimeConfig struct {
  Name string
  Path string
  ...
}

func NewRuntime(config *RuntimeConfig) (*Runtime, error) {
  r := &Runtime{
    config: config,
  }
  return r, nil
}
  ...
}
```

(sidenote, what's the point in `error` as a second return type, if errors can never occur?)